packages/framework/src/Controller/Admin/ErrorController.php

```diff
@@ -1,188 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Shopsys\FrameworkBundle\Controller\Admin;
-
-use Shopsys\FrameworkBundle\Component\Domain\Domain;
-use Shopsys\FrameworkBundle\Component\Domain\Exception\UnableToResolveDomainException;
-use Shopsys\FrameworkBundle\Component\Environment\EnvironmentType;
-use Shopsys\FrameworkBundle\Component\Error\ErrorPagesFacade;
-use Shopsys\FrameworkBundle\Component\Error\ExceptionListener;
-use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
-use Symfony\Component\ErrorHandler\Exception\FlattenException;
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\Routing\Annotation\Route;
-use Throwable;
-use Tracy\BlueScreen;
-use Tracy\Debugger;
-
-class ErrorController extends AbstractController
```

Comment:
can you please check why the preview of the error page is not working in the dev mode? 
https://symfony.com/doc/current/controller/error_pages.html#testing-error-pages-during-development

Reply:
added that feature :)